css/main.css

```diff
@@ -0,0 +1,95 @@
+.main-box{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin: auto;
+}
+
+.head-content{
+    margin: auto auto auto auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 60%;
+    margin: auto;
+    margin-top: 12rem;
+   
+
+}
+
+.head-content span{
+    color: var(--primary-color);
+    font-family:cursive;
+}
+.head-content h1{
+    font-size: 6rem;
+    margin-bottom:1.8rem;
+    text-align: center;
+    width: 95%;
+    font-weight: 900;
+    color: var(--text-color);
+    font-family:fantasy 
```

Comment:
A `;` missing after `font-family:fantasy` You could fix this.

Reply:
Fixed this